cluster-autoscaler/cloudprovider/gce/autoscaling_gke_client.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	gke "google.golang.org/api/container/v1"
+	gke_beta "google.golang.org/api/container/v1beta1"
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
+)
+
+// TODO(krzysztof-jastrzebski): Move to main.go.
```

Comment:
I would consider removing this TODO. Specifically for this flag it's not bad that it's in a less visible place (it's useless for everyone except GKE developers).

Reply:
Done